src/Adapter/Platform/Mysql.php

```diff
@@ -58,14 +57,13 @@ public function setDriver($driver)
         if ($driver instanceof Mysqli\Mysqli
             || ($driver instanceof Pdo\Pdo && $driver->getDatabasePlatformName() == 'Mysql')
             || ($driver instanceof \mysqli)
-            || ($driver instanceof \PDO && $driver->getAttribute(\PDO::ATTR_DRIVER_NAME) == 'mysql')
```

Comment:
I am afraid we cannot remove it here, as it is BC Break.
Any chance we just keep it here but not use?